keps/sig-network/1669-graceful-termination-local-external-traffic-policy/README.md

```diff
@@ -141,6 +149,266 @@ This would either happen if a version of the control plane was not aware of the
 
 There's not much risk involved as the worse case scenario is falling back to existing behavior.
 
+## Production Readiness Review Questionnaire
+
+### Feature Enablement and Rollback
+
+###### How can this feature be enabled / disabled in a live cluster?
+
+- [X] Feature gate (also fill in values in `kep.yaml`)
+  - Feature gate name: ProxyTerminatingEndpoints
+  - Components depending on the feature gate: kube-proxy
+
+###### Does enabling the feature change any default behavior?
+
+Yes, when externalTrafficPolicy=Local and there are only terimating endpoints,
+kube-proxy will route traffic to those endpoints. Before this change, kube-proxy
+dropped this traffic instead.
+
+###### Can the feature be disabled once it has been enabled (i.e. can we roll back the enablement)?
+
+Yes.
+
+###### What happens if we reenable the feature if it was previously rolled back?
+
+kube-proxy will drop traffic if only terminating endpoints are available.
+
+###### Are there any tests for feature enablement/disablement?
+
+Yes, there will be unit tests in kube-proxy with the feature gate enabled and disabled.
```

Comment:
Those sound like tests that check if the feature work.

The question is about "enanblement/disablement" process.
https://github.com/kubernetes/kubernetes/blob/release-1.21/pkg/registry/networking/networkpolicy/strategy_test.go#L246
is a nice example of how this can be tested (at least at the strategy level)

[Please say "no - they will be added" - there aren't tests now....]

Reply:
There won't be strategy tests for this feature, it is only scoped to kube-proxy.